scenarios/detection/misc/labelbox_annotation_conversion/README.md

```diff
@@ -0,0 +1,540 @@
+# Labelbox Annotation Conversion #
```

Comment:
Looks like you copied this directly from your own notes. In it's current form this isn't user-friendly or well integrated. At the bare minimum the code should be encapsulated as functions under utils_cv/detection/data.py, and a quick example/explanation provided how to call these functions (this could be in your 02 and/or 03 notebook, with the main bluk of the text however being in data.py). The json files, if you need them (for testing or as an example), could live under /data/misc since there's only two of them and they are both small

Reply:
Moved the scripts into functions in `utils_cv/detection/data.py`.  However, since the script needs to download the mask images from Labelbox, so testing may be a little bit tricky.

Reply:
Good point, I see the "urlopen" in the one of the functions. In that case maybe we only test the other function, the keypoint one does not seem to download anything from Labelbox.

Reply:
I am trying to make several dummy images into cvbp blob.  Then testing on the dummy images would make sense. :smile:

Reply:
Unit tests added